### PR TITLE
PPC0019: Add some clarifying words to explain how escapes are handled

### PR DESCRIPTION
**Pull request by leonerd**

### CODE REVIEW

File: ppcs/ppc0019-qt-string.md [hunk 1]
```diff
@@ -70,6 +70,38 @@ will be equivalent to
 qq{A ${ \scalar(TEXT) } B};
 ```
 
+Escape sequences that are recognised in `qq` strings as standing in place for
+other literal characters (such as `\n` for linefeed or `\x00` for a NUL byte)
```
Comment (mauke) [inline ppcs/ppc0019-qt-string.md:74]: I'm not sure I like calling `\n` a "linefeed". Do we do that in other places, too?

(Perl borrows `\n` from C's I/O model, which has no linefeed, only newline.)

File: ppcs/ppc0019-qt-string.md [hunk 2]
```diff
@@ -70,6 +70,38 @@ will be equivalent to
...
+
+Likewise, the escape sequences `\L`, `\l`, `\U`, `\u`, `\Q` and `\E`, which in
+`qq` strings control the way that variables are interpolated, are also not
+considered special here.  It would be helpful to users to at least issue a
+warning in this case, to remind them of this fact.
+
+```
+say qt{This does not uppercase the variable \U{ $var }};
+
+Use of the \U escape sequence is not considered special by qt quoting at FILE line LINE.
```
Comment (mauke) [inline ppcs/ppc0019-qt-string.md:102]: Why not reuse the existing message?
```
$ perl -wE '"" =~ tr/\U//'
Unrecognized escape \U passed through at -e line 1.
```

File: ppcs/ppc0019-qt-string.md (hunk 1 above)
Comment (Ovid) [reply ppcs/ppc0019-qt-string.md:74]: I would reword this to avoid the passive voice. This puts the most important information at the front.

> `qt` also recognizes the escape sequences used in `qq` strings as representing
other literal characters (such as `\n` for newline or `\x00` for a NUL byte)

File: ppcs/ppc0019-qt-string.md [hunk 3]
```diff
@@ -70,6 +70,38 @@ will be equivalent to
 qq{A ${ \scalar(TEXT) } B};
 ```
 
+Escape sequences that are recognised in `qq` strings as standing in place for
+other literal characters (such as `\n` for linefeed or `\x00` for a NUL byte)
+are also recognised by `qt`.
+
+```perl
+print qt{This message ends in a linefeed\n};
```
Comment (Ovid) [inline ppcs/ppc0019-qt-string.md:78]: `print qt{This message ends in a newline\n};`

File: ppcs/ppc0019-qt-string.md (hunk 1 above)
Comment (leonerd) [reply ppcs/ppc0019-qt-string.md:74]: Ahyes, sloppy wording on my part there. perlop(1) calls it "newline" so I should do the same.

File: ppcs/ppc0019-qt-string.md (hunk 2 above)
Comment (leonerd) [reply ppcs/ppc0019-qt-string.md:102]: Oh that's true; I forgot there was already precident there. Might as well reüse it yes.

**Comment by leonerd** [comment]
Thanks for comments all. Reworded and force-pushed

**Comment by leonerd** [comment]
Any comment all? If not I shall discuss this on PSC call tomorrow and hopefully merge then.

**Review by haarg — APPROVED** [review]

**Review by book — APPROVED** [review]
